docs/twin/microservices/gateway.md

```diff
@@ -1,16 +1,16 @@
-# OPC Gateway service (Preview)
```

Comment:
Use "services" and "modules" as folder under "docs" as discussed and remove the entire "twin" folder.   In serviecs, put everything that was in twin, except for what relates to "modules".  In docs folder, keep what is there today, plus add the architecture.md and codestructure md.  :-)